extensions/landing-zone/variables.tf

```
@@ -1,48 +1,48 @@
 variable "ibmcloud_api_key" {
   type        = string
-  description = "APIkey associated with the account to use"
+  description = "APIkey associated with the account to use."
   sensitive   = true
 }
 
 variable "region" {
   type        = string
-  description = "Region to provision all resources created by this solution"
-  default     = "au-syd"
+  description = "Region where the target IBM Cloud OpenShift Cluster is deployed."
+  nullable    = false
 }
 
 variable "cluster_id" {
   type        = string
-  description = "Id of the target IBM Cloud OpenShift Cluster"
+  description = "Id of the target IBM Cloud OpenShift Cluster."
   nullable    = false
 }
 
 variable "operator_helm_release_namespace" {
   type        = string
-  description = "Namespace to deploy the helm releases. Default to liberty-operator-helm-release"
+  description = "Namespace to deploy the helm releases. Default to liberty-operator-helm-release."
   default     = "liberty-operator-helm-release"
 }
 
 variable "add_ibm_operator_catalog" {
   type        = bool
-  description = "Flag to configure the IBM Operator Catalog in the cluster before installing the WebSphere Liberty Operator. Default is true"
+  description = "Flag to configure the IBM Operator Catalog in the cluster before installing the WebSphere Liberty Operator. Default is true."
   default     = true
 }
 
 variable "create_ws_liberty_operator_namespace" {
   type        = bool
-  description = "Flag to create the namespace where to deploy the WebSphere Liberty Operator. Default to false"
+  description = "Flag to create the namespace where to deploy the WebSphere Liberty Operator. Default to false."
   default     = false
 }
 
 variable "ws_liberty_operator_namespace" {
   type        = string
-  description = "Namespace to install the WebSphere Liberty Operator. Default to openshift-operators"
+  description = "Namespace to install the IBM WebSphere Liberty operator. Default to openshift-operators."
   default     = "openshift-operators"
 }
 
 variable "ws_liberty_operator_target_namespace" {
   type        = string
-  description = "Namespace to be watched by the WebSphere Liberty Operator. Default to null (operator to watch all namespaces)"
+  description = "Namespace to be watched by the IBM WebSphere Liberty operator. Default to null (operator to watch all namespaces)."
```

```suggestion
  description = "Namespace that the the IBM WebSphere Liberty operator watches. Default is `null`, which means that the operator watches all the namespaces."
```

This didn't get updated from an earlier review.